src/Tickets/Commerce/Models/Order_Model.php

```diff
@@ -45,8 +44,6 @@ protected function build_properties( $filter ) {
 			$gateway_slug        = Arr::get( $post_meta, [ Order::$gateway_meta_key, 0 ] );
 			$gateway_order_id    = Arr::get( $post_meta, [ Order::$gateway_order_id_meta_key, 0 ] );
 			$gateway_payload     = $this->get_gateway_payloads( $post_meta );
-			$gateway             = tribe( Manager::class )->get_gateway_by_key( $gateway_slug );
-			$gateway_name        = $gateway ? $gateway::get_label() : ucwords( str_replace( '_', ' ', $gateway_slug ) );
```

Comment:
I will suggest to have the gateway name included from the Order Model as we can re-use this in other places if needed: https://github.com/the-events-calendar/event-tickets/pull/2735/files#diff-2a52eb3e077e150d63a4b0cc3e1ba88c713f66a1499cdf6e4d4b3b8e848a57e2R31

Reply:
This is what I was saying that we should avoid, adding complexity to the Order Model.
We either pull the gateway object by default or we pull just the slug and let the specific code pull the object if they need.

Reply:
Getting the Gateway name is a simple call, too:
```$gateway_name = tribe( TEC\Tickets\Commerce\Order::class )->get_gateway_label( $order );```

Reply:
I just wasn't aware it existed

Reply:
@bordoni for me, it makes more sense to have the core info bundled into Order Model data as much as we can. Unless we are making expensive queries.

Reply:
I agree that having the whole $gateway object makes more sense, instead of having the properties seprately.

Reply:
My only concern in general is that this model is not using JIT and Caching yet, which makes each operation that we bind to the model more expensive.